tensorboard/loader_test.py

```diff
@@ -228,7 +228,10 @@ def testBigNumberInAmerica_showsCommas(self):
     self.assertIn(u' 1,024B/s ', self.bars[0])
 
   def testBigNumberInGermany_showsCommas(self):
-    locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    try:
+      locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    except Exception:
+      raise self.skipTest('Environment does not support de_DE.utf8 locale')
```

Comment:
(This…will work, but it's easier just to write `self.skipTest`, which raises the exception itself. :-) )